ipams/builtin/builtin_windows.go

```diff
@@ -13,6 +13,11 @@ import (
 	windowsipam "github.com/docker/libnetwork/ipams/windowsipam"
 )
 
+var (
+	// defaultAddressPool Store defaultAddressPool locally to use it during IPAM utils init
```

Comment:
this comment doesn't make sense.  
// defaultAddressPool is the default address pool